app/views/wiki/_wikis.html.erb

```diff
@@ -1,35 +1,39 @@
-<% if local_assigns[:digest] and wikis.size == 0 %>
-  <div class="alert alert-warning" role="alert">
-    You haven't subscribed to any topic yet. Visit <a href="https://publiclab.org/subscriptions">https://publiclab.org/subscriptions</a> to subscribe to more topics.
-  </div>
-<% else %>
-
-  <table class="table">
-    <tr>
-      <th><a href = "?sort=title"> <%= translation('wiki._wikis.title') %></a> <i class="fa fa-sort" aria-hidden="true"></i></th>
-      <th>URL</th>
-      <th><a href = "?sort=last_edited"> <%= translation('wiki._wikis.last_edited') %></a> <i class="fa fa-sort" aria-hidden="true"></i></th>
-      <th><a href = "?sort=edits"> <%= translation('wiki._wikis.edits') %></a> <i class="fa fa-sort" aria-hidden="true"></i></th>
-      <th><a href = "?sort=page_views"> <%= translation('wiki._wikis.page_views') %></a> <i class="fa fa-sort" aria-hidden="true"></i></th>
-      <th><a href = "?sort=likes"> <%= translation('wiki._wikis.likes') %></a> <i class="fa fa-sort" aria-hidden="true"></i></th>
-    </tr>
-    <% wikis.each do |wiki| %>
-      <tr>
-        <td style="width:50%"><i class="fa fa-file"></i> <a href="<%= wiki.path %>"><%= wiki.latest.title %></a></td>
-        <td><i><%= wiki.path %></i></td>
-        <td><%= distance_of_time_in_words(Time.at(wiki.latest.created_at), Time.current, { include_seconds: false, scope: 'datetime.time_ago_in_words' }) %> <%= raw translation('wiki._wikis.by') %>
-        <a href="/profile/<%= wiki.latest.author&.name %>"><%= wiki.latest.author&.name %></a></td>
-        <td><%= wiki.revisions.size %></td>
-        <td><%= number_with_delimiter(wiki.views) %></td>
-        <td><%= number_with_delimiter(wiki.cached_likes) %></td>
-      </tr>
+<hr id="first" style="border: none;height: 1px;background-color: #e2e2e2;margin-bottom: 32px;">
+<div style="font-family: sans-serif; max-width: 756px;">
+  <% @wikis.each do |wiki| %>
+    <div class="d-flex flex-column flex-md-row justify-content-between">
+      <div>
+        <div style="margin-bottom: 10px">
+          <h4 style="font-weight:500;margin:0;">
+            <a href="<%= @base_url=%><%= wiki.path %>"><%= wiki.latest.title %></a>
+          </h3>
+        </div>
+        <div style="float:left;">
+          <% if wiki.latest.author.photo? %>
+            <img style="width:20px;margin-right:8px;border-radius:50%;"src="<%= @base_url %><%= wiki.latest.author.photo_path(:thumb) %>" />
+          <% else %>
+            <img style="width:20px;margin-right:8px;border-radius:50%;"src="https://www.gravatar.com/avatar/1aedb8d9dc4751e229a335e371db8058" />
+          <% end %>
+        </div>
+        <div style="margin-bottom: 20px;">
+          <b style="font-weight: 500;"><%= wiki.latest.author.username.capitalize %></b>
+          <p style="margin: 0;color: #999;font-weight: 500;">Published <%= wiki.latest.created_at.strftime("%B %-d %Y") %></p>
+        </div>
+      </div>
+      <div>
+        <% if wiki.main_image.present? %>
```

Comment:
I think the issue with that test is that it's trying to display a pdf file as an image.
`ERROR ScreenshotsTest#test_wiki (116.34s)
Minitest::UnexpectedError:         ActionController::RoutingError: No route matches [GET] "/public/system/images/photos/250/665/868/default/test.pdf"`

So what you need to do is add a condition to this if statement so that it only allows images/gifs to be displayed.
```suggestion
        <% if wiki.main_image.present? && wiki.main_image.is_image? %>
```

Reply:
@TildaDares that sounds very promising! Are you able to run system tests locally? I have tried several times reading multiple articles, trying to fix it but with no luck. I tried on Gitpod also and the exact same errors come. One way is obviously that we push the changes and check if it works on remote. I'll do as you say 😄

Reply:
@mittal-parth You can run systems tests by running `bin/rails test:system`.

Reply:
@TildaDares what a bullseye! 🎯 That fixed the issue and tests are running fine now!